### PR TITLE
Cleanup process emulator build fix

### PR DESCRIPTION
**Pull request by cbaron**
See #8 

This should fix the build

### CODE REVIEW

File: packages/dynamodb-emulator/client.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        headers: {
+          'Content-Length': 0,
+        },
+      },
+      res => {
+        res.setEncoding('utf8');
+        const buffers = [];
+        res.once('error', reject);
+        res.on('data', buffer => {
+          if (typeof buffer === 'string') {
```
Comment (cbaron) [inline packages/dynamodb-emulator/client.js]: Not sure how @cbaron was getting a string, but jlal was not?

File: packages/dynamodb-emulator/client.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        socketPath: unixSocket,
+        timeout: requestTimeout,
+        method: 'GET',
+        path: `/${urlpath}?${qs.stringify(params)}`,
+        headers: {
+          'Content-Length': 0,
+        },
+      },
+      res => {
+        res.setEncoding('utf8');
```
Comment (lightsofapollo) [inline packages/dynamodb-emulator/client.js]: set encoding should cause everything to return a string right ?

File: packages/dynamodb-emulator/deamon.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+  try {
+    emu = await globalEmulator;
+  } catch (err) {
+    if (err) {
+      log.error(err);
+      server.close();
+    }
+  }
+  resolvedGlobalEmulator = emu;
+  console.log('emu', emu);
```
Comment (lightsofapollo) [inline packages/dynamodb-emulator/deamon.js]: might want to clean up these logs.

**Comment by lightsofapollo** [comment]
@cbaron LGTM! Can you fix the console.logs in a follow up PR ?

File: packages/dynamodb-emulator/client.js (hunk 2 above)
Comment (cbaron) [reply packages/dynamodb-emulator/client.js]: 🤦‍♂️ 👍

**Pull request merged**